docs/proposals/docker-registry-2-4.md

```diff
@@ -0,0 +1,57 @@
+# Support for Docker Registry 2.4
+## Abstract
+
+A proposal to update the docker/distribution to v2.4.0.
+
+## Motivation
+
+Primary goal is to migrate to Schema 2 [1].
```

Comment:
@miminar Also upstream have a (Vacuum)[https://github.com/docker/distribution/blob/master/registry/storage/vacuum.go] for that. But we need a StorageDriver to use it.